ninja-core/src/test/java/ninja/RouteBuilderImplTest.java

```diff
@@ -39,83 +41,69 @@
     Injector injector;
 
     @Test
-    public void testBasicGETRoute() {
-
+    public void basicGETRoute() {
```

Comment:
Regarding coding style - I guess all testcases up to now used the "test" prefix... You want to drop that - right? I am ok with that - just wanted to ask...